Integrations/AWS_DynamoDB/AWS_DynamoDB.py

```diff
@@ -0,0 +1,1697 @@
+                    config=config
+                )
+            else:
+                client = boto3.client(
+                    service_name=service,
+                    region_name=AWS_DEFAULT_REGION,
+                    aws_access_key_id=sts_response['Credentials']['AccessKeyId'],
+                    aws_secret_access_key=sts_response['Credentials']['SecretAccessKey'],
+                    aws_session_token=sts_response['Credentials']['SessionToken'],
+                    verify=VERIFY_CERTIFICATE,
+                    config=config
+                )
+    elif AWS_ACCESS_KEY_ID and AWS_ROLE_ARN:
+        sts_client = boto3.client(
+            service_name='sts',
+            aws_access_key_id=AWS_ACCESS_KEY_ID,
+            aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+            verify=VERIFY_CERTIFICATE,
+            config=config
+        )
+        kwargs.update({
+            'RoleArn': AWS_ROLE_ARN,
+            'RoleSessionName': AWS_ROLE_SESSION_NAME,
+        })
+        sts_response = sts_client.assume_role(**kwargs)
+        client = boto3.client(
+            service_name=service,
+            region_name=AWS_DEFAULT_REGION,
+            aws_access_key_id=sts_response['Credentials']['AccessKeyId'],
+            aws_secret_access_key=sts_response['Credentials']['SecretAccessKey'],
+            aws_session_token=sts_response['Credentials']['SessionToken'],
+            verify=VERIFY_CERTIFICATE,
+            config=config
+        )
+    else:
+        if region is not None:
+            client = boto3.client(
+                service_name=service,
+                region_name=region,
+                aws_access_key_id=AWS_ACCESS_KEY_ID,
+                aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+                verify=VERIFY_CERTIFICATE,
+                config=config
+            )
+        else:
+            client = boto3.client(
+                service_name=service,
+                region_name=AWS_DEFAULT_REGION,
+                aws_access_key_id=AWS_ACCESS_KEY_ID,
+                aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+                verify=VERIFY_CERTIFICATE,
+                config=config
+            )
+
+    return client
+
+
+def batch_get_item_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    kwargs = {
+        "RequestItems": json.loads(args.get("request_items", "{}")),
+        "ReturnConsumedCapacity": args.get("return_consumed_capacity", None)
+    }
+    kwargs = remove_empty_elements(kwargs)
+    if args.get('raw_json') is not None and not kwargs:
+        del kwargs
+        kwargs = safe_load_json(args.get('raw_json', "{ }"))
+    elif args.get('raw_json') is not None and kwargs:
+        return_error("Please remove other arguments before using 'raw-json'.")
```

Comment:
instead of returning error we should raise ValueError and `raw-json` => `raw_json`